decidim-assemblies/app/constraints/decidim/assemblies/current_assembly.rb

```diff
@@ -30,7 +30,13 @@ def current_assembly(env, params)
       end
 
       def detect_current_assembly(params)
-        OrganizationAssemblies.new(@organization).query.find_by_id(params["assembly_id"])
+        organization_assemblies.where(slug: params["assembly_slug"]).or(
+          organization_assemblies.where(id: params["assembly_id"])
+        ).first!
```

Comment:
I think this is what confused me... Do we need the `assembly_id` part? I tried and the redirection seems to work fine without it.

Reply:
I'll check it, sure!